test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -12480,6 +12481,58 @@ def forward(self, x):
 
         self.run_test(LerpModel(), torch.rand(5, 4, 3))
 
+    @skipScriptTest()
+    @skipIfUnsupportedMinOpsetVersion(16)
+    @unittest.skipIf(not _check_module_exists("torch_geometric"), "torch_geometric not installed.")
+    def test_sage_conv(self):
+        from torch.nn import (
+            Linear as Lin,
+            Sequential as Seq,
+            ReLU,
+            Sigmoid,
+        )
+        from torch_geometric.nn import (
+            BatchNorm,
+            knn_graph,
+            SAGEConv,
+        )
```

Comment:
Avoid importing functions. Only import modules. https://github.com/pytorch/pytorch/wiki/torch.onnx-Namespacing

Reply:
This is narrowed down to a function namespace. It doesn't pollute API neither cause circular imports

Reply:
That's great, still: unqualified names are harder to read because it's harder to know where they come from. If we always import modules, we can assume that the unqualified ones are defined within the same module. Also importing inside a function is discouraged. When possible always import on the top, unless there are dependency/ performance concerns.